compute/cker/include/cker/operation/Reduce.h

```diff
@@ -39,19 +39,47 @@ inline bool ReduceImpl(const In *input_data, const Shape &input_shape, const Sha
   const auto input_num_dims = input_shape.DimensionsCount();
 
   // Reset input iterator.
-  for (int idx = 0; idx < input_num_dims; ++idx)
+  if (num_axis == 1 && (axis[0] == -1 || axis[0] == input_num_dims - 1))
```

Comment:
`axis[0] == -1` is needless because `ResolveAxis()` makes `-1` to `input_num_dims - 1`.